### DISCUSSION TITLE
Feature request - Support for creating a release via app

### DISCUSSION THREAD

**Discussion by tejuamirthi** [community/community#33] [Mobile]
Right now, I’m not able to draft a release via mobile. Many things are absent in app like the contributors list, release details and so on

Even the mobile browser(tried on safari, chrome ios), I cannot draft for a new release. It’d be great if we could have an option to draft a release at least at from one place in mobile. We don’t do much in general apart from writing notes for the release. So any comments on this one?

**Comment by eliperkins** [comment]
@tejuamirthi thanks for the feedback here! We've definitely heard from a lot of users that they'd love to see GitHub Releases in the mobile apps. We hear ya.

What are your main use-cases for creating new Releases from within the app? We'd love to hear more about this workflow!

What are the most important things to see for Release details while on the go?

**Reply by tejuamirthi** [reply]
@eliperkins thank you for replying! Thanks for hearing too.

My use-case is that, recently when had issues with my laptop, so reviewing things via mobile only for many things! That’s really great.  Whenever I’m creating a new release, I generally give it a version and tag, also some notes describing the changes.  Github does the rest for me in creating the binaries. I think may be it’s same for most people and most cases as well.

So, this thing about a new release, which doesn’t involve actual writing code and stuff, we are creating tags, notes and version number. That sounds pretty much like a no machine kind of thing. So imagine using my iPad/iPhone/tablet and drafting a new release with the current available version of the codebase! This is going to be really exciting as maintainers it could give the extra opportunity, I feel. Let me know your thoughts, if there are barriers we can go through a work around as well to achieve the same. 



For the details part, first thing to good to have would be the number of releases, and the latest release details. If this is possible I don’t mind going a bit further and providing an option to view all the previous releases as well. I guess many things could be reused there.

**Reply by SuperFola** [reply]
Being able to draft a release on the go, edit it on your phone, delete it if you changed your mind (or made a mistake) would be useful to me.

I often find myself working on something, then remember I have to go to work and I can't tag a new release on my phone or at work until I'm back home (being very forgetful I know I won't draft said release when coming back home, unless someone reminds me to).

To me that's just another feature to bring the mobile app to parity with the website.

**Reply by iaalm** [reply]
I need this feature too. Most time, I approve a PR from someone else and I just want to create a new release to trigger deployment or something.
Currently I use browser to do it on web. I would say it's the only thing I need to use web version on mobile. Nice to see it in App.

**Reply by arharp** [reply]
I would also love to see support for creating a release in the app for the same reasons mentioned by @iaalm.

**Reply by urakozz** [reply]
My main issue right now is that I can't open the repository in the browser from the mobile phone since it redirects to the app. And in the app I can't create a release.

So I can't roll out a hotfix by just creating a new tag from my phone.

I'd appreciate having a choice instead of the force redirects to the app.

UPD: thx to Android I managed to disable app opening on links

**Reply by adityachadha-tomtom** [reply]
A specific usecase I have:

Background: our repos have Renovate integrated so that it creates pull requests and runs the corresponding test workflows. We can then look at the test outcomes and the proposed dependency upgrades and merge to master on the go which deploys stuff to CI. 

The productionization of a change requires the creation of a tag on the main branch. We do this by creating a new release whose associated tag on main triggers the production workflow. The default release notes generated using the "Generate Release Notes" works well for us, sometimes we do edit the generated notes. Doing this from the app would bring down our time to production significantly, as long as we want to keep human decision to put something to production in the process.

**Reply by kvij** [reply]
Similar use case as @adityachadha-tomtom security fixes by automation or merged PR need to be released soon. We should not need to bring a laptop to do this on holiday.

**Reply by csant** [reply]
Same as @iaalm - at time I just need to publish a new release to trigger deployment when on the run. If I could create a new release (draft) from scratch, that would be an added bonus. (and it's a pain to disable links opening in apps just to be able to open the web page in the mobile browser in order to just publish a draft).

**Reply by dandavson** [reply]
I also just need to tag a release so that my ci will run. Would be super useful

**Reply by dennemark** [reply]
I would just like to set a drafted release to latest, so that it can be published while on the go. Sometimes its just about timing.

**Reply by Shane32** [reply]
My CI workflows publish the website to development upon merge and to production upon release. This creates a clear history of what was pushed to production and when. Now with AI agents, it's feasible to use the GitHub mobile app to add a new feature, review the code the agent has written, confirm that CI tests pass, merge it in from the app, and test the feature on the dev site from my phone. The only thing missing is creating a release to push the feature to production, and in can close the loop all from my phone.

**Comment by Bettyadye8** [comment]
> Right now, I’m not able to draft a release via mobile. Many things are absent in app like the contributors list, release details and so on
> 
> Even the mobile browser(tried on safari, chrome ios), I cannot draft for a new release. It’d be great if we could have an option to draft a release at least at from one place in mobile. We don’t do much in general apart from writing notes for the release. So any comments on this one?
